packages/db/src/translate/async_operations.ts

```diff
@@ -38,7 +39,7 @@ export const translateApiAsyncOperationToPostgresAsyncOperation = (
   record: ApiAsyncOperation
 ): PostgresAsyncOperation => {
   // fix for old implementation of async-operation output assignment
-  const translatedRecord = <PostgresAsyncOperation>toSnake(record);
+  const translatedRecord = <PostgresAsyncOperation>toSnake(omit(record, 'timestamp'));
```

Comment:
Was this causing a failure? I'm assuming `timestamp` doesn't exist on the pg table for async operations.